docs/source/cpp/overview.rst

```diff
@@ -66,6 +66,9 @@ reference.
 **Kernels** are specialized computation functions running in a loop over a
 given set of datums representing input and output parameters to the functions.
 
+**Acero** (pronounced [aˈsɜɾo] / ah-SERR-oh) is a streaming execution engine that allows
```

Comment:
```suggestion
**Acero** (pronounced [aˈsɜɹo] / ah-SERR-oh) is a streaming execution engine that allows
```

This is fantastic, thanks! 

One very very minor update to the r sound in IPA. /ɾ/ is a flap (what ts and ds are frequently reduced to), and the original /r/ is common in quick transcriptions of English, but not technically accurate, since it's a trilled r like in (many dialects of) Spanish.

Everything else looks good to me!